.travis.yml

```diff
@@ -10,6 +10,24 @@ matrix:
     env: PUPPET_GEM_VERSION="~> 3.8.0" LEGACY=yes
   - rvm: 2.1.9
     env: PUPPET_GEM_VERSION="~> 4.9" DEPLOY_CANDIDATE=yes
+  - sudo: required
+    services: docker
+    rvm: '2.3.3'
+    env: PUPPET_INSTALL_VERSION="1.5.2" PUPPET_INSTALL_TYPE=agent BEAKER_set="centos-6-x86_64-docker"
```

Comment:
1.5.2 refers to the version of the puppet-agent? This is fairly old now: https://docs.puppet.com/puppet/4.10/about_agent.html#release-contents-of-puppet-agent-1x